openrtb_ext/imp_appnexus_Keywords_test.go

```diff
@@ -0,0 +1,35 @@
+package openrtb_ext
+
```

Comment:
I'd suggest just calling this file `imp_appnexus_test.go`. The test names should specify what they are specifically testing

Reply:
Addressed in [1de4912](https://github.com/prebid/prebid-server/pull/2649/commits/1de491201d6d66db96e3250ff28560cd3801e930)